h/static/scripts/group-forms/utils/annotation-metadata.ts

```diff
@@ -0,0 +1,56 @@
+import type { APIAnnotationData, PageSelector, TextQuoteSelector } from './api';
+
+/**
+ * Return the text quote that an annotation refers to.
+ */
+export function quote(annotation: APIAnnotationData): string | null {
+  if (annotation.target.length === 0) {
+    return null;
+  }
+  const target = annotation.target[0];
+  if (!target.selector) {
+    return null;
+  }
+  const quoteSel = target.selector.find(
+    (s: any) => s.type === 'TextQuoteSelector',
+  ) as TextQuoteSelector | undefined;
+  return quoteSel ? quoteSel.exact : null;
+}
+
+/**
+ * Return the label of the page that an annotation comes from.
+ *
+ * This is usually a 1-based page number, but can also be roman numerals etc.
+ */
+export function pageLabel(annotation: APIAnnotationData): string | undefined {
+  const pageSel = annotation.target[0]?.selector?.find(
+    (s: any) => s.type === 'PageSelector',
+  ) as PageSelector | undefined;
+  return pageSel?.label;
+}
+/**
```

Comment:
```suggestion
}

/**
```

Add a blank line between functions.